solutions/usaco-1206.mdx

```diff
@@ -0,0 +1,119 @@
+---
+id: usaco-1206
+source: USACO Silver 2022 February
+title: Redistributing Gifts
+author: Chongtian Ma
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_prob1_silver_feb22.html)
+
+## Explanation
```

Comment:
Don't talk in first person. Talk in third person like all editorials do (change I to we).

Reply:
fixed in the explanation part, but i kept the I in my implementation expalanation